driver-core/build.gradle

```diff
@@ -16,7 +16,7 @@
 
 
 plugins {
-    id 'de.fuerstenau.buildconfig' version '1.1.8'
+    id 'com.github.gmazzo.buildconfig' version '3.0.3'
```

Comment:
`de.fuerstenau.buildconfig` doesn't support Java 17. Found a replacement that does.